docs/samples/git-setup/clustertemplates/version_4.Y/sno-ran-du/sno-ran-du-v1.yaml

```diff
@@ -0,0 +1,289 @@
+apiVersion: o2ims.provisioning.oran.org/v1alpha1
+kind: ClusterTemplate
+metadata:
+  name: sno-ran-du.v1
+  namespace: sno-ran-du-v4-16
+spec:
+  name: sno-ran-du
+  version: v1
+  templates:
+    hwTemplate: sno-ran-du-template-configmap-v1
+    clusterInstanceDefaults: sno-ran-du-clusterinstance-defaults-v1
+    policyTemplateDefaults: sno-ran-du-policytemplate-defaults-v1
+  templateParameterSchema:
+    properties:
+      nodeClusterName:
+        type: string
+      oCloudSiteId:
+        type: string
+      policyTemplateParameters:
+        description: policyTemplateSchema defines the available parameters for cluster configuration
+        properties:
+          sriov-network-vlan-1:
+            type: string
+          sriov-network-vlan-2:
+            type: string
+          sriov-network-pfNames-1:
+            type: string
+          sriov-network-pfNames-2:
+            type: string
+          cpu-isolated:
+            type: string
+          cpu-reserved:
+            type: string
+          hugepages-default:
+            type: string
+          hugepages-size:
+            type: string
+          hugepages-count:
+            type: string
+          install-plan-approval:
+            type: string
+            default: Automatic
+      clusterInstanceParameters:
+        description: ClusterInstanceSpec defines the params that are allowed in the ClusterRequest spec.clusterInstanceInput
```

Comment:
Update to:
clusterInstanceParameters defines the available parameters for cluster deployment (or do we say installation?)

Reply:
Maybe `provisioning`?